app/src/main/java/com/nutomic/syncthingandroid/service/EventProcessor.java

```diff
@@ -145,6 +160,12 @@ public void onEvent(Event event) {
 
                 notify(title, pi);
                 break;
+            case "FolderScanProgress":
+                break;
+            case "FolderSummary":
+                Log.v(TAG, "Unhandled event " + event.type);
+                // ToDo
```

Comment:
This should trigger `updateFolderStatus(RestApi)`, or rip the status out of the event and use that.
I understand that this won't necessarily happen in this PR.

Reply:
You're right. We should do this when I got more event handling implemented as we currently poll the folder db stats every second in FoldersAdapter.java. When this permanent banging of the db REST API got replaced, we can save resources by listening to the FolderSummary event. I'll add a comment in place.